### ISSUE TITLE
tileSources type `image` not working in IE 8

### ISSUE THREAD

**Issue by zakharov-aa** [openseadragon/openseadragon#1040]
Example [image tile source](http://openseadragon.github.io/examples/tilesource-image/)

Definition:

```
OpenSeadragon({
    ...
    tileSources: {
        type: 'image',
        url:  '/example-images/grand-canyon-landscape-overlooking.jpg'
    }
    ...
});
```

Showing only black page in IE 8 Windows XP machine.

![openseadragonbugie8](https://cloud.githubusercontent.com/assets/11497853/18862095/a9c76b06-8492-11e6-9a6a-72f773638b13.png)

**Comment by avandecreme** [comment]
Yes, this tile source is not supported on IE8 because it only works with the canvas API.
Maybe we should make that clear in the doc.

**Comment by zakharov-aa** [comment]
@avandecreme  The thing is it works in all other cases.
I face this problem only in example with `type: 'image'`
The other cases works fine.
I make quick fix for this case, maby i should make pull request?

**Comment by avandecreme** [comment]
If you have a fix, yes please share :)
This wasn't supported because when we develop new features we do no take the time to support IE8, but I believe @iangilman is still accepting patches for IE8.

**Comment by zakharov-aa** [comment]
Ok, i create pull request. That's my first time so i hope i did it right. Tell me if did something wrong.

**Comment by iangilman** [comment]
Fixed by #1041
